Source/XCDBLD/SDK.swift

```diff
@@ -192,6 +212,33 @@ extension SDK {
 			}
 }
 
+extension SDK {
+    public var productName: String {
+        switch variant {
+        case .none: return self.name
+        case .macCatalyst: return "\(self.name)-catalyst"
+        }
+    }
+}
+
+extension SDK {
+    public var destination: String? {
+        switch (self.rawValue, self.variant) {
+        case (_, .macCatalyst?): return "generic/platform=macOS,variant=Mac Catalyst"
+        case ("macos", _): return "generic/platform=macOS"
```

Comment:
This is causing an error later when building for macOS
```
CarthageKit/Xcode.swift:1046: Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

I believe it should be:

```suggestion
        case ("macosx", _): return "generic/platform=macOS"
```

Reply:
Works for me otherwise, and solves my issue (see https://github.com/getsentry/sentry-cocoa/issues/2031)

Will be great to see this merged in.  Thanks for you hard work!